docs/privacy_policy.md

```diff
@@ -0,0 +1,50 @@
+### Polityka Prywatności oraz informacja o przetwarzaniu danych osobowych
+
+**Tożsamość administratora**
+
+1. Administratorem Państwa danych osobowych w rozumieniu Rozporządzenia Parlamentu Europejskiego i Rady (UE) 2016/679 UE z dnia 27 kwietnia 2016 r. (dalej: „RODO”), po zarejestrowaniu w serwisie oraz w związku z korzystaniem z jego funkcjonalności jest Fundacja Code for Poznań mająca siedibę w... zarejestrowaną w...
+
+**Dane kontaktowe administratora**
+
+2. Z administratorem można się skontaktować poprzez adres email *administrator@codeforpoznan.pl* lub pisemnie na adres siedziby administratora.
+
+**Dane kontaktowe inspektora ochrony danych osobowych**
+
+3. Administrator wyznaczył inspektora ochrony danych, z którym można skontaktować poprzez email *inspektor@codeforpoznan.pl*.
+
+**Cele przetwarzania oraz podstawy prawne przetwarzania (!)**
+
+4. Państwa dane osobowe będą przetwarzane przez administratora (i podmioty powiązane - WRK???), w szczególności w następujących celach:
+
+zawarcia i realizacji umowy o świadczenie usług drogą elektroniczną na warunkach opisanych w „Regulaminie Portalu”,
+umożliwienia kontaktu z administratorem za pośrednictwem formularzy kontaktowych,
+analitycznym i statystycznym,
+
+na podstawie art. 6 ust. 1 lit. f RODO, tj. w oparciu o niezbędność przetwarzania do celów wynikających z prawnie uzasadnionych interesów realizowanych przez Administratora lub przez stronę trzecią,
+prawnie uzasadnione interesy:
+wysyłanie korespondencji e-mail dotyczącej działań Fundacji,
+rejestracja użytkownika,
+administrator nie przetwarza szczególnych kategorii danych osobowych ani danych osobowych dotyczących wyroków skazujących oraz naruszeń prawa lub powiązanych środków bezpieczeństwa,
```

Comment:
W kontekście obowiązku informowania o tym, co się przetwarza brak wzmianki o przetwarzaniu np. szczególnych kategorii danych osobowych (na tyle na ile pozwala mi moja orientacja prawna) oznacza że tych danych się nie przetwarza, także nie wiem czy istnieje konieczność informowania o tym, czego się nie przetwarza. Ale to tylko refleksja - zostawiam otwarte do dyskusji :)